lm75bd/lm75bd.c

```diff
@@ -26,9 +26,41 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
+
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  //sets the temp register
+  error_code_t errCode;
+
+  if ( temp == NULL){
+    return ERR_CODE_INVALID_ARG;
+  }
+  uint8_t buff[1] = {0};
```

Comment:
Make sure the register address is defined as a macro for readability purposes. Look at LM75BD_REG_CONF macro as an example on how it's done.